inlong-audit/audit-sdk/src/main/java/org/apache/inlong/audit/IpPortListLoader.java

```diff
@@ -0,0 +1,33 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements. See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.inlong.audit;
+
+import java.util.List;
+import java.util.Map;
+
+/**
+ * IpPortListLoader
+ */
+public interface IpPortListLoader {
+
+    String KEY_IPPORT_LOADER = "audit.ipport.loader";
```

Comment:
`ipport` is not a word, how about use "audit.ip.port.loader" or "audit.address.loader"

Reply:
Fixed